scripts/specializations/CpAIWorker.lua

```diff
@@ -504,4 +505,33 @@ function CpAIWorker:onStartAutoDrive()
         SpecializationUtil.raiseEvent(self, "onCpADStartedByPlayer")
         CpJobStartAtLastWpSyncRequestEvent.sendEvent(self)
     end
-end
\ No newline at end of file
+end
+
+---------------------------------------------
+--- Console commands
+---------------------------------------------
+
+function CpAIWorker.registerConsoleCommands()
+    g_devHelper.consoleCommands:registerConsoleCommand("cpVehicleOnWorkStartTest", 
+        "Raise work start command", "consoleCommandRaiseWorkStart", CpAIWorker)
```

Comment:
It is a bit confusing what this is doing especially it seems to raise an event (not raise the implements) which lowers the implements. Also, only foldable controllers are notified, so its scope seems to be the foldables only, what about the other controllers?

Reply:
Forgot to comment my intentions there, was to acting to fast ...
For now I only needed the foldable controller for testing and the other could be added later when needed.